eslint-plugin/docs/rules/avoid-high-accuracy-geolocation.md

```diff
@@ -0,0 +1,48 @@
+# Alerts to not use high accuracy geolocation when trying to locate a user (`@ecocode/avoid-high-accuracy-geolocation`)
+
+⚠️ This rule _warns_ in the ✅ `recommended` config.
+
+<!-- end auto-generated rule header -->
+
+## Rule details
+
+This rule aims at reducing CPU consumption by telling the device to use a less accurate yet more eco friendly geolocation, when geolocation API is used.
+
+## Examples
+
+Examples of **incorrect** code for this rule:
```

Comment:
As requested in another PR, I think we can use "non compliant"/"compliant" in this file, instead of "incorrect"/"correct"